builds/installing-openshift-builds.adoc

```diff
@@ -27,7 +27,7 @@ include::modules/ob-installing-builds-using-CLI.adoc[leveloffset=+1]
 include::modules/ob-creating-a-shipwright-build-resource-cli.adoc[leveloffset=+2]
 
 
+[role="_additional-resources"]
+== Additional resources
 
-// [role="_additional-resources"]
-// == Additional resources
-
+* You can learn more about installing Operators on {product-title} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-adding-operators-to-a-cluster[Adding Operators to a cluster] section.
```

Comment:
Nit: See the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) about formatting additional resources lists.